private/bufpkg/bufplugin/bufplugin_test.go

```diff
@@ -57,7 +57,7 @@ func TestPluginRegistryRoundTrip(t *testing.T) {
 	})
 	assertPluginRegistryRoundTrip(t, &bufpluginconfig.RegistryConfig{
 		NPM: &bufpluginconfig.NPMRegistryConfig{
-			ImportStyle: "module",
+			ImportStyle: "commonjs",
```

Comment:
wasn't sure if this was changed intentionally?

Reply:
Yeah sorry meant to add a comment. Noticed a branch with no test coverage - by changing this it was exercised.